src/codegen/utilities/enum_helpers.py

```diff
@@ -0,0 +1,192 @@
+import html
+import logging
+import re
+
+from html.parser import HTMLParser
+_logger = logging.getLogger(__name__)
+_logger.addHandler(logging.NullHandler())
+
+
+# We don't need this stuff.
+ENUMS_BLACKLIST = [
+    "AltRef",
+    "AntStatus",
+    "DAQmxErrors",
+    "DAQmxWarnings",
+    "ExportActions",
+    "GroupBy",
+    "ForceIEPEUnits",
+    "FilterType1",
+    "LengthUnits4",
+    "NavMeasurementType",
+    "NavMode",
+    "SaveOptions",
+    "TaskControlAction",
+    "Timescale",
+    "WatchdogAOOutputType",
+    "WatchdogControlAction",
+    "VelocityUnits2",
+]
+
+
+# Metadata issues or invalid Python names (leading number)
+NAME_SUBSTITUTIONS = {
+    '100_MHZ_TIMEBASE': 'ONE_HUNDRED_MHZ_TIMEBASE',
+    '20_MHZ_TIMEBASE': 'TWENTY_MHZ_TIMEBASE',
+    '2_POINT_5_V': 'TWO_POINT_FIVE_V',
```

Comment:
Do any attribute/function docstrings reference these values? If so, it would be better to do these substitutions in scrapigen so that the docstrings have the substituted values.

Reply:
The `NAME_SUBSTITUTIONS` were not used in docstrings of attributes/functions.